src/main/java/redis/clients/jedis/JedisPubSub.java

```diff
@@ -15,11 +15,9 @@
 import redis.clients.jedis.exceptions.JedisException;
 import redis.clients.util.SafeEncoder;
 
-public abstract class JedisPubSub {
+public abstract class JedisPubSub extends BinaryJedisPubSub {
```

Comment:
I think this breaks binary compatibility but I'm not entirely sure, @HeartSaVioR ?. If that is the case, this PR can only go into Jedis 3.0